server/src/concepts/user/useCases/signInUser.js

```diff
@@ -0,0 +1,26 @@
+import User from "../model/User.js";
+import bcrypt from "bcrypt";
+import { generateToken } from "../../authorization/controllers/auth.js";
+
+export const signInUserFunc = async (email, password) => {
+  if (!email || !password) {
+    throw new Error("Need email and password");
+  }
+  const user = await User.findOne({ email: email })
+    .select("email password")
+    .exec();
+
+  if (!user) {
+    throw new Error("User not found");
```

Comment:
```suggestion
    throw new Error("Invalid email or password");
```
It is better to throw error without specific info about what it wrong with authentication.